lib/lita/adapters/slack/message_handler.rb

```diff
@@ -124,7 +124,7 @@ def channel
           data["channel"]
         end
 
-        def dispatch_message(user)
+        def dispatch_me_message(user)
```

Comment:
Can you please add a comment with the url your showed me when we were pairing for the me_message type? I was confused about the naming, didn't realize it was a message type.

Reply:
Yep! Here are the docs https://api.slack.com/methods/chat.meMessage